analyzer/evmtokens/evm_tokens.go

```diff
@@ -190,10 +190,12 @@ func (m Main) processBatch(ctx context.Context) (int, error) {
 				if err != nil {
 					return fmt.Errorf("downloading mutated token %s: %w", staleToken.Addr, err)
 				}
-				batch.Queue(m.qf.RuntimeEVMTokenUpdateQuery(),
-					staleToken.Addr,
-					mutable.TotalSupply.String(),
-				)
+				if mutable != nil {
```

Comment:
does it make sense to add an error log here if `mutable` is nil?

Reply:
https://github.com/oasisprotocol/oasis-indexer/blob/78c85ac6f16f7fcfd8662a684e39c90d0414e450/analyzer/modules/evm.go#L140 it logs deep within :grimacing: